lighthouse-core/gather/driver.js

```diff
@@ -142,6 +150,13 @@ class Driver {
     return this.evaluateAsync(`(${pageFunctions.ultradumbBenchmarkString})()`);
   }
 
+  /**
+   * @param {number} timeout
+   */
+  setProtocolTimeout(timeout) {
+    this._protocolTimeout = timeout;
```

Comment:
I afraid we'll need to make timeout a setting on `sendCommand` with a global default :/

Reply:
We can move to that if it's really needed. It complicates calling it quite a bit. And due to the unique typing on the method, I think the best option would be this:

```
sendCommand(methodOrOpts, ...params) {
  const {method, timeout} = typeof methodOrOpts === 'object' ? methodOrOpts : {
    method,
    timeout: DEFAULT_PROTOCOL_TIMEOUT
  }
}
```

That way, the calling code for most use cases can remain the same.